### ISSUE TITLE
Quickstart Step #2 

### ISSUE THREAD

**Issue by ghost** [YahooArchive/mojito#846]
Greetings,

Found a small issue on the Quickstart Guide. It is concerning testing whether or not Mojito was installed correctly. See below:
1.    Get Mojito from the npm registry and globally install Mojito so that it can be run from the command line. You may need to use sudo if you run into permission errors.
   
   $ npm install mojito -g
2.    Confirm that Mojito has been installed by running unit tests.
   
   $ mojito test

Result of Step 2)

oshi@ubuntu:~/Projects/mojito/hello_world$ mojito test
path.existsSync is now called `fs.existsSync`.
⚠ Disabled. Without arguments, this command no longer runs mojito's unit tests

is there another way of verifying this?

Thanks,
Oshi

**Comment by rwaldura** [comment]
Oshi: yes, but it's more involved, as we now depend on Arrow to run the tests. 
https://github.com/yahoo/arrow
We'll have to expand the documentation to include how to do this.
For now, I'd proceed assuming the installation succeeded.

**Comment by zhouyaoji** [comment]
This has been fixed in the source file for the documentation: https://github.com/yahoo/mojito/blob/develop/docs/dev_guide/quickstart/index.rst

I replaced 'mojito test' with 'mojito help' to confirm that Mojito has been installed. The Mojito documentation on YDN will be updated after some more changes have been made for the next release. Thanks for filing the issue. I'll close the issue once the docs have been updated on YDN.
